src/lib/operators/jit_operator/operators/jit_write_tuples.cpp

```diff
@@ -60,19 +61,36 @@ void JitWriteTuples::_create_output_chunk(JitRuntimeContext& context) const {
     const auto is_nullable = output_column.tuple_value.is_nullable();
 
     // Create the appropriate column writer for the output column
-    resolve_data_type(data_type, [&](auto type) {
-      using ColumnDataType = typename decltype(type)::type;
-      auto column = std::make_shared<ValueColumn<ColumnDataType>>(output_column.tuple_value.is_nullable());
+    if (data_type == DataType::Bool) {
+      // Data type bool is a special jit data type and not an opossum data type
+      // Therefore, bools needs to be handled separately as they are transformed to opossum int types
+      auto column = std::make_shared<ValueColumn<int32_t>>(output_column.tuple_value.is_nullable());
```

Comment:
Yeah, there are way too many places where we assume that we use an `int32_t` for that. If we don't already have it, please introduce a type for this.